### PR TITLE
Fix #665, improve figures

### LINKED ISSUE #665: Improve figures
|Version|Source|Clause(s)|
|:---:|:---:|:---:|
|AOM Working Group Draft, 17 July 2023|Apple|all|

#### Comment:
All figures should follow the same styling (e.g. same font, same colours, etc.)

Figure 1: Contrast issues, text in green blocks too dark or the colour of the boxes too dark. Increase contrast. Caption does not mention that this is an example
Figure 2: Text size is too small. Codec Enc. Codec Dec. almost invisible because of contrast issues. (Terms Codec Enc. / Dec. should also be improved (see related issue))

Figure 3: Small text, contrast, styling
Figure 4: contrast issues, formula symbols
Figure 5: resolution issues,
Figure 6: should be a table
Figure 7: Contrast issues, text hard to read
Figure 8: Gray background in the text, not using same styling
Figure 9: Text placement can be improved
Figure 10: Text placement can be improved. Also text too large (make sure to keep same font size across figures whenever possible)
Figure 11: same
Figure 13 - 16:  should be rather formulas (is it possible to use MathML?)
Figure 17: Use same styles, gray background in some of the text
Figure 18: Text placement
Figure 19: text too small
Figure 20: text too small, resolution issues
Figure 21: same
Figure 22: Enc. vs Encs. does not look good in the specification. Please fix.
Figure 23: could be a formula as well. MathML?

-----
#### Suggestion:
Improve all figures across the spec.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/727.html" title="Last updated on Aug 29, 2023, 10:34 PM UTC (1de520f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/727/92797d4...1de520f.html" title="Last updated on Aug 29, 2023, 10:34 PM UTC (1de520f)">Diff</a>

### CODE REVIEW

**Comment by tdaede** [comment]
@felicialim will re-add comments.

File: images/Ambisonics Decoding Flowchart.png
Comment (felicialim) [inline images/Ambisonics Decoding Flowchart.png:1]: How about renaming the "Channel Mapping / Demixing Matrix" module in the green box to be "Ambisonics Reconstruction", to distinguish it from the same terms used next to the arrow.

File: images/Channel Audio Decoding Flowchart.png
Comment (felicialim) [inline images/Channel Audio Decoding Flowchart.png:1]: Output_Gain -> "output_gain" to be consistent with the text
Recon_Gain could be e.g. Recon Gain (different from the recon_gain, to distinguish it from the data flow next to the arrows)

File: images/Down-mix Parameter and Loudness.png
Comment (felicialim) [inline images/Down-mix Parameter and Loudness.png:1]: down-mixed audio -> Down-mixed audio

File: images/IA Decoder Configuration.png
Comment (felicialim) [inline images/IA Decoder Configuration.png:1]: "Mix_Presentation" implies a syntax but it's not a valid syntax. Is it referring to the metadata to be applied? If so, how about saying "Apply Mix Presentation metadata" instead?

File: images/IA Encoder Configuration.png
Comment (felicialim) [inline images/IA Encoder Configuration.png:1]: "Ambix mode" -> "Ambisonics mode"
"Enc. #2" -> Encoder #2

File: images/IA Encoding Flowchart for Channel Audio Format.png
Comment (felicialim) [inline images/IA Encoding Flowchart for Channel Audio Format.png:1]: PCMs -> PCM
CH -> CG
Output_Gain -> output_gain, same with Dmixp_Mode and Recon_Gain. To be consistent with the text
Prefer "Loudness level 1" to be on 1 line
How about making "Channel Audio" font size the same as other blocks?

File: images/IAMF Timing Model.png
Comment (felicialim) [inline images/IAMF Timing Model.png:1]: If possible, avoid "Sequence of PBOs" and "OBU Packetizing" from overlapping the border/arrow underneath them.

File: images/ISOBMFF Trimming Handling.png
Comment (felicialim) [inline images/ISOBMFF Trimming Handling.png:1]: PTS && Audio Samples -> PTS & Audio Samples

File: index.bs [hunk 1]
```diff
@@ -1588,15 +1588,29 @@ class ReconGainInfoParameterData() {
...
+     of      :      b3      : Left surround channel (or Lss)
+the 1st byte :      b4      : Right surround channel (or Rss)
+             :      b5      : Left top front channel 
+             :      b6      : Right top front channel
+----------------------------------------------------------             
+ LSB 5 bits  :      b7      : Left rear surround channel
+ of the 2nd  :      b8      : Right rear surround channel
+  byte when  :      b9      : Left top back channel
+ MSB of the  :      b10     : Right top back channel
+1st byte = 1 :  b11 (MSB)   : Low-Frequence Effect channel
```
Comment (felicialim) [inline index.bs:1607]: Low-Frequence -> Low-Frequency

File: index.bs [hunk 2]
```diff
@@ -1588,15 +1588,29 @@ class ReconGainInfoParameterData() {
...
 <b>Semantics</b>
 
-<dfn noexport>recon_gain_flags</dfn> is a bitmask that indicates which channels [=recon_gain=] is applied to, as depicted in the figure below.
+<dfn noexport>recon_gain_flags</dfn> is a bitmask that indicates which channels [=recon_gain=] is applied to, as shown in the table below.
 
-<left><img src="images/Recon_Gain_Flags.png" style="width:100%; height:auto;"></left>
-<center><figcaption>Table for Recon Gain Flags</figcaption></center>
+<pre class = "def">
+Byte postion : Bit position : Assigned Channel Name
+             :   b0 (LSB)   : Left channel)
```
Comment (felicialim) [inline index.bs:1595]: Left channel) -> Left channel

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [inline index.bs:1607]:
```suggestion
1st byte = 1 :  b11 (MSB)   : Low-Frequency Effect channel
```

File: index.bs (hunk 2 above)
Comment (sunghee-hwang) [inline index.bs:1595]:
```suggestion
             :   b0 (LSB)   : Left channel
```

**Comment by sunghee-hwang** [comment]
> lgtm % question about what CH stands for in images/IA Encoding Flowchart for Channel Audio Format.png. Should it be CG?

It stands for Channel.

**Comment by felicialim** [comment]
> > lgtm % question about what CH stands for in images/IA Encoding Flowchart for Channel Audio Format.png. Should it be CG?
> 
> It stands for Channel.

Thanks for clarifying. I suggest either writing out "channel" in the figure, or adding it to the figure legend.

**Pull request merged**